web/src/components/lineage/components/node/Node.tsx

```diff
@@ -43,25 +46,52 @@ const Node: React.FC<NodeProps> = ({ node, selectedNode, setSelectedNode }) => {
     return '/'
   }
 
+  // configure the tool tip
+  const HtmlTooltip = styled(({ className, ...props }: TooltipProps) => (
```

Comment:
I think we want this to be in a different file by itself called `MQTooltip.tsx` to allow for component reuse.